tests/func/test_ignore.py

```diff
@@ -66,3 +68,49 @@ def test_ignore_in_parent_dir(self):
         all_paths = self._get_all_paths()
 
         self.assertNotIn(forbidden_path, all_paths)
+
+
+def test_metadata_unchanged_when_moving_ignored_file(dvc_repo, repo_dir):
```

Comment:
Do we check that dir mtime is changed when non-ignored file moved without modification?

Reply:
Added test, also for deletion